ext/storage_sources/s3_store/test/test_s3_connection.cpp

```diff
@@ -35,7 +35,11 @@ const Aws::String region = Aws::Region::AP_SOUTHEAST_2;
 const double throughputTargetGbps = 5;
 const uint64_t partSize = 8 * 1024 * 1024;  // 8 MB.
 static std::string bucketName("s3testext"); // Can be overridden with environment variables.
-static std::string objPrefix("s3test_artefacts--unit_"); // To be concatenated with a random string.
+
+// Objects with the prefex pattern "s3test/*" are deleted after a certain period of time according
+// to the lifecycle rule on the S3 bucket. Should you wish to make any changes to the prefix pattern
+// or lifecycle of the object, please speak to the release manager.
+static std::string objPrefix("s3test/"); // To be concatenated with a random string.
```

Comment:
Should this be `s3test/unit/`?

Reply:
I considered this, it does make sense to match the pattern more closely to the python tests. I will add this in now!